Squish allows (#2084)
n8n-io/n8n-docs: docs/code/builtin/output-other-nodes.md

	| `$("<node-name>").last(branchIndex?, run Index?)` | The last item output by the given node. | :white_check_mark: |
	| `$("<node-name>").item` | The linked item. This is the item in the specified node used to produce the current item. Refer to [Item linking](/data/data-mapping/data-item-linking/) for more information on item linking. | :x: |
	| `$("<node-name>").params` | Object containing the query settings of the given node. This includes data such as the operation it ran, result limits, and so on. | :white_check_mark: |

> 	| `$("<node-name>").context` | Only available when working with the Loop Over Items node. Provides information about what's happening in the node, helping you determine whether the node is still processing items. | :white_check_mark: |

Reviewer: ```suggestion
	| `$("<node-name>").context` | Boolean. Only available when working with the Loop Over Items node. Provides information about what's happening in the node. Use this to determine whether the node is still processing items. | :white_check_mark: |
```

	| `_("<node-name>").last(branchIndex?, run Index?)` | The last item output by the given node. | :white_check_mark: |
	| `_("<node-name>").item` | The linked item. This is the item in the specified node used to produce the current item. Refer to [Item linking](/data/data-mapping/data-item-linking/) for more information on item linking. | :x: |
	| `_("<node-name>").params` | Object containing the query settings of the given node. This includes data such as the operation it ran, result limits, and so on. | :white_check_mark: |

> 	| `_("<node-name>").context` | Only available when working with the Loop Over Items node. Provides information about what's happening in the node, helping you determine whether the node is still processing items. | :white_check_mark: |

Reviewer: ```suggestion
	| `_("<node-name>").context` | Boolean. Only available when working with the Loop Over Items node. Provides information about what's happening in the node. Use this to determine whether the node is still processing items. | :white_check_mark: |
```
